### ISSUE TITLE
[iOS] Screen spinning wildly during navigation

### ISSUE THREAD

**Issue by sanketgarade** [organicmaps/organicmaps#3256]
Version: ⁦2022.08.23-3⁩
iOS 15.6.1

Today during navigation the screen was spinning very randomly as you can see in the video. This is my first time doing navigation using OM in from the app itself (not CarPlay).

**Comment by biodranik** [comment]
Wow, interesting. It would be great to stable reproduce it.

**Comment by sanketgarade** [comment]
Yes I'll try to reproduce it and share here. I don't have my car for a few days so will try in next week.

**Comment by biodranik** [comment]
Please enable logs in OM settings and remember the exact time when it happens, so we can later debug logs (you can share them via "?" -> Report a bug).

**Comment by sanketgarade** [comment]
Sorry but where is "enable logs"? I can't see it in the settings. See screenshot. 

![image](https://user-images.githubusercontent.com/74632379/187056217-ca83c047-1b49-4a7d-82b0-e36779904e1b.png)

**Comment by biodranik** [comment]
Sorry, my mistake, logs are available for Android version only.
